tests/console/check_system_info.pm

```diff
@@ -105,9 +105,36 @@ sub check_python2_dropped {
     record_info('python2 dropped but still can be searched', 'Bug 1196533 - Python2 package still can be searched after migration to SLES15SP4', result => 'fail') if $out;
 }
 
+# SLE-23610: Python3 module
+# test steps:
+# 1) activate the python3 module
+# 2) install the python310 package
+# 3) check python3.10's version which should be 3.10.X
+# 4) check python3's version
+# 5) check python310's lifecycle
+# 6) remove python310 package
```

Comment:
I still not understand why we need remove it, it impact other test cases?

Reply:
it's just the cleanup process for the test,  it'll not impact the rest test till now.